src/components/SummaryPageCard/SummaryPageCard.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { ReactElement } from "react";
+import { SummaryPageCardProps } from "./SummaryPageCardTypes";
+import Button from "../Button";
+import {
+	BottomActionWrapper,
+	ContentWrapper,
+	RecentActivityWrapper,
+	TitleWrapper,
+	TitleBar,
+} from "./SummaryPageCard.styled";
+
+const RecentActity = (props: SummaryPageCardProps): ReactElement => {
+	const {
+		bottomAction,
+		content,
+		size = "lg",
+		title,
+		titleIcon,
+		topAction,
+	} = props;
+
+	return (
+		<RecentActivityWrapper>
+			<TitleBar size={size}>
+				<TitleWrapper>
+					{titleIcon}
+					{title}
+				</TitleWrapper>
+				{topAction && (
+					<Button {...topAction} muiAttrs={{ disableRipple: true }}></Button>
```

Comment:
We should not be disabling the ripple anymore because we want users to be able to navigate the website with just the tab, and the ripple let's them know which button they're highlighting